docs/creating_service_module.md

```diff
@@ -0,0 +1,55 @@
+## Creating a new Service Module in this repository
+
+This section walks through adding a new Service Module project to the SDK repository.
+
+### Creating the project
+
+New Service Modules in this repository should be created under the `org.aerogear.mobile.*` namespace.
+
+See the [`keycloak-service-module`](../keycloak-service-module/build.gradle) for examples.
+
+1. Via Android Studio
```

Comment:
Make this a fouth level header instead of a list.  This way we can link people to the AS or CLI instructions as appropriate.  Otherwise +1